### PR TITLE
Add Synapse connected buckets for data proc jobs

### PR DESCRIPTION
**Pull request by ahayden**
- [x] This change makes uat/prod buckets connected to synapse and owned by Synapse service accounts. These are the for the output of data processing jobs that will run in scicomp, including for app migration and for R proc
- [x] Passes pre-commit

### CODE REVIEW

File: config/prod/dian-prod-synapse-import-s3.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  AllowWriteBucket: "true"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # THIS CURRENTLY ONLY WORKS IN THE us-east-1 REGION!!!!!
+  # Data transfers within the same region between AWS resources are free.
+  # true to restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda) within the same region as this bucket.
+  # This will not allow even the owner of the bucket to download objects in this bucket when not using an AWS resource in the same region!
+  SameRegionResourceAccessToBucket: "true"
```
Comment (zaro0508) [inline config/prod/dian-prod-synapse-import-s3.yaml:25]: Enabling this only make sense if you `GrantAccess` to users that might download files from regions outside of us-east-1.  Wondering why you are enabling this when the only thing that has access is synapse?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/prod/dian-prod-synapse-import-s3.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+template_path: "remote/s3-bucket-v2.j2"
+stack_name: "dian-prod-synapse-import-s3"
+sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  SynapseIDs: ["3425758"]  #synapse-service-dian-prod@synapse.org
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "DIAN"
```
Comment (zaro0508) [inline config/prod/dian-prod-synapse-import-s3.yaml:12]: hmm, the Platform dept has a DIAN project?

File: config/prod/dian-prod-synapse-import-s3.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  # LifecycleDataStorageClass: 'INTELLIGENT_TIERING'
+  # (Optional) Number of days until S3 objects are moved to the LifecycleDataStorageClass, default is 30
+  # LifecycleDataTransition: '90'
+  # (Optional) Number of days (from creation) when objects are deleted from S3 and LifecycleDataStorageClass, default is 365000
+  # LifecycleDataExpiration: '1825'
+
+# For CI system (do not change)
+hooks:
+  before_launch:
+    - !cmd "wget https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/s3-bucket-v2.j2 -N -P templates/remote"
```
Comment (zaro0508) [inline config/prod/dian-prod-synapse-import-s3.yaml:46]: i would recommend pinning to the latest tag.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Comment by ahayden** [comment]
Closing as will be superseded by PR based on new templates

**Pull request closed without merging**